utilities/transactions/transaction_test.h

```diff
@@ -68,7 +68,7 @@ class TransactionTestBase : public ::testing::Test {
     options.two_write_queues = two_write_queue;
     dbname = test::PerThreadDBPath("transaction_testdb");
 
-    DestroyDB(dbname, options);
+    EXPECT_OK(DestroyDB(dbname, options));
```

Comment:
There should be an EXPECT_OK at line 88 (instead of assert)

Reply:
Done.